src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs

```diff
@@ -319,36 +319,6 @@ private async Task BlazorWasmHostedTemplate_IndividualAuth_Works(BrowserKind bro
             }
         }
 
-        [Theory]
-        [InlineData(BrowserKind.Chromium, Skip = "https://github.com/dotnet/aspnetcore/issues/28596")]
-        public async Task BlazorWasmStandaloneTemplate_IndividualAuth_Works(BrowserKind browserKind)
```

Comment:
Could you clarify why we're removing this test case?

Reply:
Because it is problematic as it doesn't use valid data and as a result the user is redirected to `https://login.microsoftonline.com`.

Reply:
Understood. Would you be able to update the referenced issue #28596 to reflect the fact that it's no longer a *flaky* test, but now is a permanent gap in our test coverage until someone creates a valid test here? Otherwise it will get confusing when we try to look at #28596.

Reply:
I've added some comments to ensure this is covered by CTI somehow.